solr.cfg

```diff
@@ -39,12 +39,13 @@ java_opts =
   -server
   -Xms${settings:solr-min-ram}
   -Xmx${settings:solr-max-ram}
+
 index =
-    name:allowedRolesAndUsers   type:string stored:false multivalued:true
```

Comment:
ic. The "default" field should be "stored=false" then though, according to the Solr docs.